doc/sphinx-guides/source/installation/config.rst

```diff
@@ -2841,6 +2871,50 @@ should delete the old JVM option and the wrapped password alias, then recreate
 as shown for :ref:`dataverse.pid.datacite.password` but with the EZID alias
 name.
 
+.. _dataverse.pid.crossref.url:
```

Comment:
These are about legacy support which is no longer implemented - they can all be removed

Reply:
removed